urllib3/__init__.py

```diff
@@ -22,11 +24,12 @@
 
 # Set default logging handler to avoid "No handler found" warnings.
 import logging
-try:  # Python 2.7+
+if sys.version_info >= (2, 7):
     from logging import NullHandler
-except ImportError:
-    class NullHandler(logging.Handler):
+else:
+    class NullHandler(logging.Handler):  # Python 2.6
         def emit(self, record):
+            # type: (logging.LogRecord) -> None
```

Comment:
It's really disappointing that we'd have to change away from EAFP to LBYL in order to get the typing module to work properly. Is there really no alternative?